docs/_documentations/mdt-vsc-commands-project.md

```diff
@@ -23,21 +23,25 @@ Right-click a project in the **Codewind view** to see most project commands. All
 
 - **Open app**: Open the project application root endpoint in the system browser. The project must be in the *Running* or *Debugging* state, or the application server won't be available.
 
-- **Open application monitor**: Open the Codewind **Application Monitor** page for this project in the system browser.
+- **Show Project Overview**: Open an editor tab that displays all of the Codewind information for the project. From this page, you can build, disable, or delete the project, and you can change the project auto build setting. This page is the only place in the extension where you can delete a project. If you delete a project, you remove it from both Codewind and from your file system.
```

Comment:
oh, just noticed! this has some outdated info.

```
This page is the only place in the extension where you can delete a project. If you delete a project, you remove it from both Codewind and from your file system. 
```
Please remove these two sentences

Reply:
Removed.